### PR TITLE
Releasing 2.0.0-beta.7 (#1175)

### PR DESCRIPTION
**Pull request by erinha**
Merging release branch back into main.

### CODE REVIEW

**Review by Jessica-Koch — COMMENTED** [review]
:)

**Review by Jessica-Koch — APPROVED** [review]

File: packages/teams-js/README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ To install the latest [2.0 preview version](https://docs.microsoft.com/en-us/jav
 
 ### Production
 
-You can access [these files on unpkg](https://res.cdn.office.net/teams-js/2.0.0-beta.6/js/MicrosoftTeams.min.js), download them, or point your package manager to them.
+You can access [these files on unpkg](https://res.cdn.office.net/teams-js/2.0.0-beta.7/js/MicrosoftTeams.min.js), download them, or point your package manager to them.
```
Comment (AE-MS) [inline packages/teams-js/README.md:27]: >unpkg [](http://example.com/codeflow?start=31&length=5)

Just noticed this in the readme. Technically these aren't on unpkg.

**Review by AE-MS — APPROVED** [review]
:shipit:

File: packages/teams-js/README.md (hunk 1 above)
Comment (erinha) [reply packages/teams-js/README.md:27]: Will fix before the next release.

**Pull request merged**